modules/integration/tests-ui-integration/src/test/java/org/wso2/carbon/mb/ui/test/topic/TopicTenantTestCase.java

```diff
@@ -0,0 +1,127 @@
+*  specific language governing permissions and limitations
+*  under the License.
+*/
+package org.wso2.carbon.mb.ui.test.topic;
+
+
+import org.eclipse.paho.client.mqttv3.MqttException;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.integration.common.utils.exceptions.AutomationUtilException;
+import org.wso2.mb.integration.common.clients.ClientMode;
+import org.wso2.mb.integration.common.clients.MQTTClientConnectionConfiguration;
+import org.wso2.mb.integration.common.clients.MQTTClientEngine;
+import org.wso2.mb.integration.common.clients.MQTTConstants;
+import org.wso2.mb.integration.common.clients.QualityOfService;
+import org.wso2.mb.integration.common.clients.exceptions.AndesClientConfigurationException;
+import org.wso2.mb.integration.common.clients.exceptions.AndesClientException;
+import org.wso2.mb.integration.common.utils.backend.MBIntegrationUiBaseTest;
+import org.wso2.mb.integration.common.utils.ui.pages.configure.AddNewTenantPage;
+import org.wso2.mb.integration.common.utils.ui.pages.configure.ConfigurePage;
+import org.wso2.mb.integration.common.utils.ui.pages.login.LoginPage;
+import org.wso2.mb.integration.common.utils.ui.pages.main.HomePage;
+
+import javax.jms.JMSException;
+import javax.naming.NamingException;
+import javax.xml.xpath.XPathExpressionException;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+
+/**
+ * UI test case(s) for tenant related topics.
+ */
+public class TopicTenantTestCase extends MBIntegrationUiBaseTest {
+
+    /**
+     * Initializes the test case.
+     *
+     * @throws AutomationUtilException
+     * @throws XPathExpressionException
+     * @throws MalformedURLException
+     */
+    @BeforeClass()
+    public void init() throws AutomationUtilException, XPathExpressionException, MalformedURLException {
+        super.init();
+    }
+
+    /**
+     * 1. Create the new tenant account
+     * 2. Publish and subscribe MQTT messages using the created tenant admin account.
+     *
+     * @throws XPathExpressionException
+     * @throws IOException
+     */
+    @Test(groups = {"wso2.mb", "queue", "mqtt"}, description = "https://wso2.org/jira/browse/MB-1180")
+    public void performAddNewTenantPublishSubscribeQueue() throws XPathExpressionException, IOException,
+            AndesClientConfigurationException, JMSException, NamingException, AndesClientException, MqttException {
+        driver.get(getLoginURL());
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = loginPage.loginAs(getCurrentUserName(), getCurrentPassword());
+
+        ConfigurePage configurePage = homePage.getConfigurePage();
+        AddNewTenantPage addNewTenantPage = configurePage.getAddNewTenantPage();
+
+        String domain = "home.com";
```

Comment:
No, the JIRA that I am referring need to create the tenant admin using the UI. Therefore not using the automation.xml